var/spack/repos/builtin/packages/googletest/package.py

```diff
@@ -67,5 +71,7 @@ def install(self, spec, prefix):
                          prefix.include)
 
             mkdirp(prefix.lib)
-            install('libgtest.a', prefix.lib)
-            install('libgtest_main.a', prefix.lib)
+            install('libgtest.{0}'.format(
+                'so' if '+shared' in spec else 'a'), prefix.lib)
```

Comment:
This needs to be changed to the appropriate macOS-compatible way.  Search the repository for examples; or maybe @davydden would know.

Reply:
Ah, thanks for pointing that out.  I'll do some research!